solr/core/src/test/org/apache/solr/schema/SpatialRPTFieldTypeTest.java

```diff
@@ -28,34 +28,34 @@
 import org.locationtech.spatial4j.shape.Shape;
 
 public class SpatialRPTFieldTypeTest extends AbstractBadConfigTestBase {
-  
+
   private static File tmpSolrHome;
   private static File tmpConfDir;
-  
+
   private static final String collection = "collection1";
   private static final String confDir = collection + "/conf";
-  
+
   @Before
   private void initManagedSchemaCore() throws Exception {
     tmpSolrHome = createTempDir().toFile();
     tmpConfDir = new File(tmpSolrHome, confDir);
     File testHomeConfDir = new File(TEST_HOME(), confDir);
     FileUtils.copyFileToDirectory(new File(testHomeConfDir, "solrconfig-managed-schema.xml"), tmpConfDir);
-    FileUtils.copyFileToDirectory(new File(testHomeConfDir, "solrconfig-basic.xml"), tmpConfDir);
+    //FileUtils.copyFileToDirectory(new File(testHomeConfDir, "solrconfig-basic.xml"), tmpConfDir);
```

Comment:
Why the added commented lines?

Reply:
so, these config files aren't used by this test.

However, maybe I should open a new JIRA about auditing config files in tests....